recipes/sphobjinv/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "sphobjinv" %}
+{% set version = "2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bskinn/sphobjinv/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 29c2287afe54682bdc6ff66337c36af7dda63b1198166bae7aa0185012e46dde
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

You are missing the entry_point here. This should fix the Windows build.

```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
  entry_points:
    - sphobjinv = sphobjinv.cli.core:main
```

---

Right, thx for catching this.  One of the issues of not using grayskull, clearly!

---

Yes and no. Grayskull does a great job at finding these in setup.py. I'm not sure if setup.cfg is supported and I know pyporject.toml is not. We are working on it though. BTW, you can use grayskull on GitHub releases.